server/report/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Try to keep the CRs for one thing at a time in the future please :)

Reply:
ye